### DISCUSSION TITLE
How to config options to show label for example <select any word> ?

### DISCUSSION THREAD

**Discussion by jqhr** [folke/flash.nvim#151] [Q&A] [answered]
For example <select any word>, if there are many same words, how to navigate to one of theme?
![图片](https://github.com/folke/flash.nvim/assets/23305067/e7d631c2-2486-47f8-9dae-4c319b2d3961)
I want to go to position with red frame, how can I do?

**Comment by folke** [comment]
There's an example in the docs to jump to a word.
Not sure what you mean with red frame

**Comment by jqhr** [comment]
![图片](https://github.com/folke/flash.nvim/assets/23305067/add1587f-07d0-4a4f-8684-cc136ba048cc)

**Reply by folke** [reply]
ok, I see what you mean. That's a bug. Let me check

**Reply by folke** [reply, marked as answer]
The example code was not correct. I've updated the docs. Use the code below:
```lua
require("flash").jump({
  pattern = ".", -- initialize pattern with any char
  search = {
    mode = function(pattern)
      -- remove leading dot
      if pattern:sub(1, 1) == "." then
        pattern = pattern:sub(2)
      end
      -- return word pattern and proper skip pattern
      return ([[\<%s\w*\>]]):format(pattern), ([[\<%s]]):format(pattern)
    end,
  },
  -- select the range
  jump = { pos = "range" },
})

**Reply by jqhr** [reply]
Thanks, it is very nice.

**Comment by gsinclair** [comment]
I just want to say it is not clear to me, when reading the docs, what that example is supposed to do. From memory, it is headlined "jump to any word". What does that mean?

**Reply by folke** [reply]
Literally jump to a word. Search pattern with word boundary
